### DISCUSSION TITLE
GITHUB_REF and GITHUB_SHA no longer work

### DISCUSSION THREAD

**Discussion by nulano** [community/community#25207] [Actions]
<p>I was working on finishing my workflow, when suddenly I started seeing nonsensical errors. It turns out GITHUB_REF and GITHUB_SHA are no longer getting set!</p>
<p>Running cmd /c “set” on Windows gives:</p>
<pre><code>[...]
GITHUB_BASE_REF=
GITHUB_EVENT_NAME=push
GITHUB_EVENT_PATH=d:\a\_temp\_github_workflow\event.json
GITHUB_HEAD_REF=
GITHUB_REF=
GITHUB_REPOSITORY=nulano/Pillow
GITHUB_SHA=
[...]
</code></pre>
<p>This broke about an hour ago.</p>
<p>Also see this issue: <a href="https://github.com/actions/checkout/issues/49" rel="nofollow noopener">https://github.com/actions/checkout/issues/49</a></p>

**Comment by ethomson** [comment, marked as answer]
<p>Thanks for letting us know, and sorry for the inconvenience.  We’re working on this now.</p>

**Comment by svenstaro** [comment]
<p>Can you post an update here once this starts working again?</p>

**Comment by timruffles** [comment]
<p>This should have been fixed now <a class="mention" href="/svenstaro">@svenstaro</a>  🙇‍♂️</p>

**Comment by svenstaro** [comment]
<p>Thanks. Just tested, it’s indeed working.</p>

**Comment by britzl** [comment]
<p>Are you sure this is working? I didn’t get any value for GITHUB_REPOSITORY today.</p>

**Comment by jednano** [comment]
<p>I also had issues with <code>GITHUB_REPOSITORY</code>, but it looks like <code>github.repository</code> might work?</p>
